examples/python/PointCloud/point_cloud_showcase.py

```
@@ -158,35 +158,37 @@ def main() -> None:
     for i, pcd in enumerate(camFrames):
-        print_point_cloud_info(pcd, i)
+        printPointCloudInfo(pcd, i)
 
     # 4 ── Custom transform + passthrough depth
-    print_header("4. Custom transform matrix + passthrough")
+    printHeader("4. Custom transform matrix + passthrough")
     print("  Config: 90° Z rotation via initialConfig")
-    for i, (pcd, depth) in enumerate(zip(customFrames, depthFrames, strict=True)):
-        print_point_cloud_info(pcd, i)
+    if len(customFrames) != len(depthFrames):
+        raise RuntimeError("customFrames and depthFrames must have the same length")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)